x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/__mocks__/es_results.ts

```diff
@@ -50,7 +61,7 @@ export const sampleDocWithSortId: SignalSourceHit = {
   _type: 'doc',
   _score: 100,
   _version: 1,
-  _id: 'someFakeId',
+  _id: uuid.v4(),
```

Comment:
For unit tests you usually want to avoid any type of non-repeatable data such as uuid or date times or auto generated test data. Any way you can just hard code a GUID here instead?